Sage/carbon
# refactor(popover): convert internal popover to typescript - FE-4862

### Proposed behaviour

This PR converts internal Popover component to TypeScript

### Checklist




#### QA



### Testing instructions

Check components that are using Popover:
- Select
- Date
- SplitButton
- MultiActionButton
- ActionPopover
